source/hassio/index.markdown

```diff
@@ -25,11 +25,11 @@ The advantages of using Hass.io:
 
 ### Upgrading
 
-Hass.io users can update Home Assistant via the 'Hass.io' panel in the UI. However, please note that Home Assistant updates take some time to roll into the Hass.io builds. Therefore, there is often a slight delay (a day or two) between the availability of a Home Assistant update and the update being available in Hass.io, so be patient. When a Hass.io update is available, it will be shown as a notification in the ‘Hass.io' panel in the web interface.
+Home Assistant users can update via the 'Supervisor' panel in the UI. However, please note that updates take some time to roll into the Home Assistant builds. Therefore, there is often a slight delay (a day or two) between the availability of a Home Assistant Core update and the update being available in Home Assistant, so be patient. When a Home Assistant update is available, it will be shown as a notification in the sidebar panel in the web interface.
```

Comment:
> Home Assistant users can update via the 'Supervisor' panel in the UI.

Updating what? That is now dropped off the sentence. 

> it will be shown as a notification in the sidebar panel in the web interface

That is actually not true and should be removed.